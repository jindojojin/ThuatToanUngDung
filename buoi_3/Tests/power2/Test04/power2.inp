10
1992000000 8 19920000 185620000 1147000000 128 256 1147000000 33554432 1073741824
