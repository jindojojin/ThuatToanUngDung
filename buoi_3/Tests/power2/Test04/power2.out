8 128 256 33554432 1073741824
